sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/optimizer/RewriteSubquerySuite.scala

```diff
@@ -52,4 +52,13 @@ class RewriteSubquerySuite extends PlanTest {
     comparePlans(optimized, correctAnswer)
   }
 
+  test("Perform propagating empty relation after RewritePredicateSubquery") {
```

Comment:
Shall we add `SPARK-31280: ` prefix?
- https://spark.apache.org/contributing.html

Reply:
>> In practice, usually it is added when a JIRA type is a bug or a PR adds a couple of tests to an existing test class

Reply:
Got it.

Reply:
I added the Jira id because I also think it is good to add in this case although it does not exactly fit these rules